### PR TITLE
Add Launchpad Manager Yosemite

### PR DESCRIPTION
**Pull request by ChristinWhite**
Add the Yosemite version of Launchpad Manager. 

Here's the homepage: http://launchpadmanager.com

I'm not 100% certain the token should be `launchpad-manager-yosemite` rather than simply `launchpad-manager`. In previous versions Launchpad Manager supported multiple OS versions but the developer is treating this as a new, stand-alone application and I followed his convention.

The app filename is `Launchpad Manager Yosemite.app`

And the upgrade notice email I received as an owner of the previous version states:

> A new OS X Yosemite compatible version of Launchpad Manager is available for download. Please note that this is not an update of the existing Launchpad Manager application, but a separate application. This version is not compatible with previous versions of OS X such as Lion, Mountain Lion and Mavericks

Also note that the download link on the homepage is to a PHP file rather than pointing directly to the DMG. I figured out the direct link through Safari's developer tools and I have tested the cask and confirmed that it's giving me the correct, most recent version.

### CODE REVIEW

File: Casks/launchpad-manager-yosemite.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'http://launchpadmanager.com/download_yosemite.php/LaunchpadManagerYosemite.dmg'
+  name 'Launchpad Manager Yosemite'
+  homepage 'http://launchpadmanager.com/'
+  license :commercial
+
+  app 'Launchpad Manager Yosemite.app'
+end
```
Comment (vitorgalvao) [inline Casks/launchpad-manager-yosemite.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/launchpad-manager-yosemite.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'launchpad-manager-yosemite' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://launchpadmanager.com/download_yosemite.php/LaunchpadManagerYosemite.dmg'
+  name 'Launchpad Manager Yosemite'
```
Comment (vitorgalvao) [inline Casks/launchpad-manager-yosemite.rb]: Should be simply “Launchpad Manager”, as “Yosemite” does not appear to be part of the branding.

**Comment by christopherdwhite** [comment]
Great, will fix and resubmit.

**Comment by vitorgalvao** [comment]
The token is fine. Since it is derived from the app bundle’s name, it is correct as is.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It will be merged in [a different pull request](https://github.com/caskroom/homebrew-cask/pull/8809) due to some simple recommended alterations. Your contribution is still included (and still credited to you), with the appropriate modifications, when applicable.

Changes:
+ Corrected `name`.

**Pull request closed without merging**